schedule-2019-03-internal.md

```diff
@@ -0,0 +1,95 @@
+# Instructor Training (Internal, March 2019)
+
+Copyright 2019, RStudio Inc.
+All material made available under the [Creative Commons Attribution-NonCommercial 4.0](https://creativecommons.org/licenses/by-nc/4.0/) license.
+
+## Instructors
+
+- Garrett Grolemund
+- Greg Wilson
+
+## Day 1
+
+1.  Getting Started (0:15/09:15)
+
+1.  Two Ways to Teach (0:45/10:00)
+    -   <./keynote/10-two-ways-to-teach.key>
+    -   Comparison with movie/play (add comedy improv)
+    -   Tell participants there's lots of research (much of which people don't know/act on)
+    -   Explain what active learning is (creates a real-time feedback loop in class)
+    -   Introduce cognitive architecture diagram (revisit it several times)
+        -   Teaching is "load into short-term memory and keep it there long enough to be transferred"
```

Comment:
I'd like to add an exercise that helps them refactor a passive RStudio teaching experience into an active one.